frontend/app/src/composables/message-handling.ts

```diff
@@ -183,6 +184,34 @@ export const useMessageHandling = () => {
     };
   };
 
+  const handleAccountingRuleConflictMessage = (
+    data: AccountingRuleConflictData
+  ): Notification | null => {
+    const { numOfConflicts } = data;
+
+    if (numOfConflicts === 0) {
+      return null;
+    }
```

Comment:
Is that a real possibility? do we get a messages with number of conflicts being 0?